### PR TITLE
Update throughput settings tab with new elasticity properties

### PR DESCRIPTION
**Pull request by vchske**
[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/1461)

Elasticity team will expose two new properties to help customers know if their change to throughput will be instant, take several hours, or will need to be handled via a support request.
Other minor improvements also made.

### CODE REVIEW

**Review by jawelton74 — APPROVED** [review]

File: src/Common/dataAccess/readCollectionOffer.ts [hunk 1]
```diff
@@ -104,6 +112,8 @@ const readCollectionOfferWithARM = async (databaseId: string, collectionId: stri
         manualThroughput: undefined,
         minimumThroughput,
         offerReplacePending: resource.offerReplacePending === "true",
+        instantMaximumThroughput,
+        softAllowedMaximumThroughput: softAllowedMaximumThroughput,
```
Comment (ato9000) [inline src/Common/dataAccess/readCollectionOffer.ts:116]: nit: consider changing to 
```
instantMaximumThroughput,
softAllowedMaximumThroughput,
```
to be consistent

File: src/Common/dataAccess/readCollectionOffer.ts [hunk 2]
```diff
@@ -113,6 +123,8 @@ const readCollectionOfferWithARM = async (databaseId: string, collectionId: stri
       manualThroughput: resource.throughput,
       minimumThroughput,
       offerReplacePending: resource.offerReplacePending === "true",
+      instantMaximumThroughput,
+      softAllowedMaximumThroughput: softAllowedMaximumThroughput,
```
Comment (ato9000) [inline src/Common/dataAccess/readCollectionOffer.ts:127]: same comment as above to be consistent

File: src/Common/dataAccess/readDatabaseOffer.ts [hunk 3]
```diff
@@ -76,6 +84,8 @@ const readDatabaseOfferWithARM = async (databaseId: string): Promise<Offer> => {
         manualThroughput: undefined,
         minimumThroughput,
         offerReplacePending: resource.offerReplacePending === "true",
+        instantMaximumThroughput,
+        softAllowedMaximumThroughput: softAllowedMaximumThroughput,
```
Comment (ato9000) [inline src/Common/dataAccess/readDatabaseOffer.ts:88]: same comment as above to be consistent

File: src/Common/dataAccess/readDatabaseOffer.ts [hunk 4]
```diff
@@ -85,6 +95,8 @@ const readDatabaseOfferWithARM = async (databaseId: string): Promise<Offer> => {
       manualThroughput: resource.throughput,
       minimumThroughput,
       offerReplacePending: resource.offerReplacePending === "true",
+      instantMaximumThroughput,
+      softAllowedMaximumThroughput: softAllowedMaximumThroughput,
```
Comment (ato9000) [inline src/Common/dataAccess/readDatabaseOffer.ts:99]: same comment as above to be consistent

**Review by ato9000 — DISMISSED** [review]

**Review by ato9000 — APPROVED** [review]

**Review by jawelton74 — APPROVED** [review]

**Pull request merged**